qa/rpc-tests/show_help.py

```diff
@@ -49,6 +49,13 @@
        Execute command when the best block changes (%s in cmd is replaced by
        block hash)
 
+|  -blocksonly
```

Comment:
Why are all these added, and why do they have leading `|` symbols?

Reply:
This is testing the output of `zcashd --help -help-debug`; see the first commit.